docs/locations.md

```diff
@@ -0,0 +1,165 @@
+# Backup Storage Locations and Volume Snapshot Locations
+
+Ark v0.10 introduces a new way of configuring where Ark backups and their associated persistent volume snapshots are stored.
+
+## Motivations
+
+In Ark prior to v0.10, the configuration for where to store backups & volume snapshots was stored in a `Config` custom resource. The `backupStorageProvider` section captured the place where all Ark backups should be stored. This was defined by a **provider** (e.g. `aws`, `azure`, `gcp`, `minio`, etc.), a **bucket**, and possibly some additional provider-specific settings (e.g. `region`). Similarly, the `persistentVolumeProvider` section captured the place where all persistent volume snapshots taken as part of Ark backups should be stored, and was defined by a **provider** and additional provider-specific settings (e.g. `region`).
```

Comment:
I agree with @wwitzel3 on the tense. Because that version is not deprecated, it would sound more natural to talk about it in the present term because it is still in use.